airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -134,6 +138,32 @@ def get_asset_events(
     )
 
 
+@assets_router.post(
+    "/events",
+    responses=create_openapi_http_exception_doc([404]),
+)
+def create_asset_events(
```

Comment:
```suggestion
def create_asset_event(
```

We are creating a single event.

Reply:
Done